content/pt/docs/guias/APROVACAO/_index.md

```diff
@@ -8,4 +8,29 @@ draft: false
 description: >
 
 ---
+<br>
 
+### Overview
```

Comment:
hey @agabiandrade , since its a second level title on this page, wouldn't be better to keep it as:

```suggestion
## Overview
```

and follow this standard in other pages? those change will be nice for SEO purposes as well

Reply:
changed!